easybuild/easyconfigs/r/R/R-3.6.0-intel-2019a.eb

```diff
@@ -0,0 +1,2195 @@
+    }),
+    ('quantreg', '5.40', {
+        'checksums': ['86e310a235009ab85635dfb8803c175f80a35892e237db2525c4ef37a98936eb'],
+    }),
+    ('robustbase', '0.93-5', {
+        'checksums': ['bde564dbd52f04ab32f9f2f9dd09b9578f3ccd2541cf5f8ff430da42a55e7f56'],
+    }),
+    ('sp', '1.3-1', {
+        'checksums': ['57988b53ba8acc35f3912d62feba4b929a0f757c6b54080c623c5d805e0cb59f'],
+    }),
+    ('zoo', '1.8-5', {
+        'checksums': ['8773969973d28d7d1a48f74b73be1dbd97acb3b22a4668a102e8bb585a7de826'],
+    }),
+    ('lmtest', '0.9-37', {
+        'checksums': ['ddc929f94bf055974832fa4a20fdd0c1eb3a84ee11f716c287936f2141d5ca0a'],
+    }),
+    ('vcd', '1.4-4', {
+        'checksums': ['a561adf120b5ce41b66e0c0c321542fcddc772eb12b3d7020d86e9cd014ce9d2'],
+    }),
+    ('snowfall', '1.84-6.1', {
+        'checksums': ['5c446df3a931e522a8b138cf1fb7ca5815cc82fcf486dbac964dcbc0690e248d'],
+    }),
+    ('rpart', '4.1-15', {
+        'checksums': ['2b8ebe0e9e11592debff893f93f5a44a6765abd0bd956b0eb1f70e9394cfae5c'],
+    }),
+    ('survival', '2.44-1.1', {
+        'checksums': ['55b151e15fcd24ccb3acf60331c9a7ad82bc10f3841ab3be9bc2a37e9ee751b9'],
+    }),
+    ('bindr', '0.1.1', {
+        'checksums': ['7c785ca77ceb3ab9282148bcecf64d1857d35f5b800531d49483622fe67505d0'],
+    }),
+    ('plogr', '0.2.0', {
+        'checksums': ['0e63ba2e1f624005fe25c67cdd403636a912e063d682eca07f2f1d65e9870d29'],
+    }),
+    ('bindrcpp', '0.2.2', {
+        'checksums': ['48130709eba9d133679a0e959e49a7b14acbce4f47c1e15c4ab46bd9e48ae467'],
+    }),
+    ('purrr', '0.3.2', {
+        'checksums': ['27c74dd9e4f6f14bf442473df22bcafc068822f7f138f0870326532f143a9a31'],
+    }),
+    ('tidyselect', '0.2.5', {
+        'checksums': ['5ce2e86230fa35cfc09aa71dcdd6e05e1554a5739c863ca354d241bfccb86c74'],
+    }),
+    ('dplyr', '0.8.1', {
+        'checksums': ['5f3ef7f0378d89799d466be254e25857a2747d70d5d97450618ac9303f1481db'],
+    }),
+    ('tidyr', '0.8.3', {
+        'checksums': ['a18f54ec35124110058ab23f7e0a3c037a8d50f0405520cf5cc5443ec022cc37'],
+    }),
+    ('mnormt', '1.5-5', {
+        'checksums': ['ff78d5f935278935f1814a69e5a913d93d6dd2ac1b5681ba86b30c6773ef64ac'],
+    }),
+    ('foreign', '0.8-71', {
+        'checksums': ['3bd2392fbcd9dd4f385ac1418a5e454b9769e6414a21fbb5ae1e7ce6072760d7'],
+    }),
+    ('psych', '1.8.12', {
+        'checksums': ['6e175e049bc1ee5b79a9e51ccafb22b962b4e6c839ce5c9cfa1ad83967037743'],
+    }),
+    ('generics', '0.0.2', {
+        'checksums': ['71b3d1b719ce89e71dd396ac8bc6aa5f1cd99bbbf03faff61dfbbee32fec6176'],
+    }),
+    ('broom', '0.5.2', {
+        'checksums': ['16af7b446b24bc14461efbda9bea1521cf738c778c5e48fcc7bad45660a4ac62'],
+    }),
+    ('nloptr', '1.2.1', {
+        'checksums': ['1f86e33ecde6c3b0d2098c47591a9cd0fa41fb973ebf5145859677492730df97'],
+    }),
+    ('boot', '1.3-22', {
+        'checksums': ['cf1f0cb1e0a7a36dcb6ae038f5d0211a0e7a009c149bc9d21acb9c58c38b4dfc'],
```

Comment:
please add alternate checksum for `boot`, see https://github.com/easybuilders/easybuild-easyconfigs/pull/8762/files#diff-3a13eccc331bec9e3aee8a5e7bd10916